source/FAQ/index.rst

```diff
@@ -50,8 +67,9 @@ helps the |CL| team proactively identify and resolve bugs. See the
 What is the default firewall?
 =============================
 
-|CL| packages :command:`iptables` as a bundle, however, there are no default
-firewall rules. All network traffic is allowed by default.
+|CL| packages :command:`iptables` and :command:`firewalld` as optional
+bundles, however, there are no default firewall rules. All network traffic is
+allowed by default.
```

Comment:
Ln 81.
Remove Ln 81: "Files under `:file: /etc/` ... and replace with:
|CL| ships with a mostly empty `/etc`, leaving configuration to the user. Read more about `user areas`_.

Add hyperlink at bottom: 
`.. _user areas: https://docs.01.org/clearlinux/latest/guides/clear/stateless.html#user-areas`

Reply:
Ln 83-85, also please embed hyperlink. 
See `this blog post`_ explaining how to customize `/etc/fstab`.
Add hyperlink at bottom: 
.. _this blog post: https://clearlinux.org/news-blogs/where-etcfstab-clear-linux